apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -91,99 +90,118 @@ const Attachment: FC = () => {
     }
   };
 
+  const preventDefaultEvent = (e: Event) => {
+    e.preventDefault();
+  };
+
   return (
-    <Menu as="div">
-      <Menu.Button
-        aria-label="More"
-        as={motion.button}
-        className="outline-brand-500 rounded-full outline-offset-8"
-        onClick={() => setShowMenu(!showMenu)}
-        whileTap={{ scale: 0.9 }}
-      >
-        {isUploading ? (
-          <Spinner size="sm" />
-        ) : (
-          <Tooltip content="Media" placement="top">
-            <PhotoIcon className="text-brand-500 size-5" />
-          </Tooltip>
-        )}
-      </Menu.Button>
-      <MenuTransition show={showMenu}>
-        <Menu.Items
-          className="absolute z-[5] mt-2 rounded-xl border bg-white py-1 shadow-sm focus:outline-none dark:border-gray-700 dark:bg-gray-900"
+    <DropdownMenuPrimitive.Root
+      modal={false}
+      onOpenChange={setShowMenu}
+      open={showMenu}
+    >
+      <div>
```

Comment:
do we need this extra div?

Reply:
Yes, This is needed to align the Picture Icon with other icons present